pkg/pub_package_reader/lib/src/names.dart

```diff
@@ -39,6 +39,7 @@ const reservedWords = <String>{
   'implements',
   'import',
   'in',
+  'inline',
```

Comment:
Does this change belong?

Reply:
It is a CI issue that needs to be fixed, either here or another one.

Reply:
removed, added new PR https://github.com/dart-lang/pub-dev/pull/6686